src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -171,3 +173,20 @@ Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+Napi::Value ModelWrap::clone(const Napi::CallbackInfo& info) {
+    std::vector<std::string> allowed_signatures;
+    try {
+        if (ov::js::validate(info, allowed_signatures)) {
+            return cpp_to_js(info.Env(), _model->clone());
+        }
+        else {
+            OPENVINO_THROW("'clone'", ov::js::get_parameters_error_msg(info, allowed_signatures));
+        }
+    }
+    catch(const std::exception& e) {
+        reportError(info.Env(), e.what());
+        return info.Env().Undefined();
+    }
+}
+
```

Comment:
```suggestion
```
This line is extra. You can use `clang_format_fix_all` target

Reply:
Done.